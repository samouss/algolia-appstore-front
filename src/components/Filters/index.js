import React from 'react';
import PropTypes from 'prop-types';
import './index.css';

const Filters = ({ children }) => (
  <div styleName="Filters">
    {children}
  </div>
);

Filters.propTypes = {
  children: PropTypes.node.isRequired,
};

export default Filters;
