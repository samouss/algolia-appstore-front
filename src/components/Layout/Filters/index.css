.Filters {
  margin-bottom: 2.5vh;
}
