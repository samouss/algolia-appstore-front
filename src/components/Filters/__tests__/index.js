import React from 'react';
import { shallow } from 'enzyme';
import Filters from '../index';

describe('<Filters />', () => {
  const defaultProps = {};

  it('expect to render', () => {
    const props = {
      ...defaultProps,
    };

    const component = shallow(
      <Filters
        {...props}
      >
        Content
      </Filters>,
    );

    expect(component).toMatchSnapshot();
  });
});
