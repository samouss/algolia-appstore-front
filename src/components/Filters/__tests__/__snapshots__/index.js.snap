// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Filters /> expect to render 1`] = `
<div
  styleName="Filters"
>
  Content
</div>
`;
